frontend/src/utils/utils.ts

```diff
@@ -1,14 +1,17 @@
 import { supportedChains, TokenInfo } from 'src/components/models';
 import { BigNumber, BigNumberish, hexValue, parseUnits, formatUnits } from './ethers';
-
 /**
  * @notice Generates the Etherscan URL based on the given `txHash` or `address and `chainId`
  */
 export const getEtherscanUrl = (txHashOrAddress: string, chainId: number) => {
-  const group = txHashOrAddress.length === 42 ? 'address' : 'tx';
+  const group = txHashOrAddress.length === 42 ? 'address' : txHashOrAddress.length === 66 ? 'tx' : 'ens';
```

Comment:
This isn't exactly correct, since you can have a 42 or 66 character ENS name. This should use `isHexString` to identify a txHash/address or the presence of a `.` means it must be an ENS name

Reply:
How's this https://github.com/ScopeLift/umbra-protocol/blob/5b7e7358a76fd8794ef53f8ea14f7c34accf0c05/frontend/src/utils/utils.ts#L7

Reply:
LGTM!

Reply:
You're amazing, thanks!!